js/popUp.js

```diff
@@ -0,0 +1,116 @@
+let y = [
+  {
+    name: 'Tonic',
+    description:'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: './images/project_screenshots/Snapshoot Portfolio.svg',
+    technologies: ['Html', 'Ruby on rails', 'css', 'javaScript'],
+    live: 'https://example.com',
+    source: 'https://github.com/EvansSnave/Mobile-design/tree/main',
+    imageS: './images/project_screenshots/Snapshoot Portfolio.svg',
+    description: `Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.`,
+    canopy: ['Full Stack Dev', 2015],
+  },
+  {
+    name: 'Multi-Post Stories',
+    description:"Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.",
+    image: './images/project_screenshots/Snapshoot Portfolio (1).svg',
+    technologies: ['Html', 'Ruby on rails', 'css', 'javaScript'],
+    live: 'https://example.com',
+    source: 'https://github.com/EvansSnave/Mobile-design/tree/main',
+    imageS: './images/project_screenshots/Snapshoot Portfolio (1).svg',
+    description: `Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.`,
+    canopy: ['Back End Dev', 2015],
+  },
+];
```

Comment:
- please i kindly suggest you to use a more decriptive name for the array of objects containing each project details :+1: 
- [ ] you have suplicate properties for images and description, please make sure to have just a single image and description for each project, that was you are implementing the `DRY` principle in your javascript code base :+1: :sparkles: 
- [ ] For each popup currently the images being displayed are not being displayed correctly and this is because each project card has its own unique image, you are required to display the same image on a project card for the popup as well, you are not to use a single image for every popup like you are currently doing at the moment. Each popup should be unique in terms of the project details from the `name, description, image and links` as provided in the array of objects which you created.
- below is the requirement for the modal :+1: 
![d-q](https://github.com/EvansSnave/Mobile-design/assets/89055502/1c89abb1-b38e-4a02-8774-5a1a778ef954)

Reply:
@Efiamotu-1 Hi. thank you for your review. I dont understand . The images displayed in the pop up window must be different from each other and the images on the work section? And you said my descriptions are duplicated but they are different I think. I don't know exactly why they are duplicated. Thank you for your time.

Reply:
@EvansSnave for each project card, when you click on `see project` to display the popup modal, the image that is displayed in the popup should be the same as the one on the card that was clicked, this is the expected behaviour but what you currently have is the image below being displayed anytime you click on either the first or second project.
![img](https://github.com/EvansSnave/Mobile-design/assets/89055502/0143fbd1-496e-47e6-a93b-74c87e5e2c23)
- [ ] for the description look at `line 15 & 21`, you have the same description.
- please let me know if anything is still not clear :+1: